MaxText/tokenizer.py

```diff
@@ -16,6 +16,8 @@
 
 """Provides op for tokenizing a dataset."""
 
+# I expect this script must be rewritten for PyTorch/XLA
+
```

Comment:
We can easily re-write this script: https://pytorch.org/text/main/transforms.html#sentencepiecetokenizer. Should be 1,2 days of work.